hedera-node/test-clients/src/main/resource/spec-default.properties

```diff
@@ -36,7 +36,7 @@ default.node=0.0.3
 default.node.name=DEFAULT_NODE
 default.nodePayment.tinyBars=5000
 default.payer=0.0.2
-recordStream.path=../data/recordstreams/record0.0.3
+recordStream.path=hedera-node/data/recordstreams/record0.0.3
```

Comment:
Is this correct? Typically tests in `test-clients` need to be run from the `test-clients` directory, not the `hedera-services` (root) directory

Reply:
It's correct, this was fixed recently - it's no longer required to execute tests from the `test-clients` working directory